### PR TITLE
About form

### PR DESCRIPTION
**Pull request by IvanLudick**
## In this section I did to following:

- The section you need to build for this milestone is: 
    - the contact form in the footer.
We included a screenshot of Template 1 for reference, but you should follow the template that you chose.
- Make sure that:
    - Name field accepts up to 30 chars.
    - Email field accepts only emails.
    - Message field accepts up to 500 chars.
    - All fields have placeholder text.
    - All fields have the correct type.
    - A user cannot submit the form if one or more of the fields is empty.
- In order to lay out the elements on the page you should use Flexbox.
- You don't need to implement any functionality that requires JavaScript.
- You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -227,6 +231,29 @@ <h1 class="about-title">About <br>Myself</h1>
...
+
+      <h2 class="form-title">Contact me</h2>
+      <div class="contact-info">
+        <p>If you have an application you are interested in developing, a feature that you need build or a project that needs coding. I'd love to help with.</p>
+      </div>
+      
+      <form class="form-action" action="https://formspree.io/f/{form_id}" method="post">
+        <input name="text" class="lable" placeholder="Yeremias" type="email" maxlength="30" required>
+        <input name="Email" class="lable" placeholder="natajayanj14@gmail.com" id="email" type="email" required>
+        <textarea name="textarea" class="text-area" placeholder="Write your message here" maxlength="500"></textarea>
```
Comment (mckent05) [inline index.html]: - According to the project requirements, all input fields should have the `required` attribute, please include this for the `textarea` 👍 👍

File: index.html [hunk 2]
```diff
@@ -227,6 +231,29 @@ <h1 class="about-title">About <br>Myself</h1>
...
+    <div class="form-bg"></div>
+
+    <div class="from-descript">
+
+      <h2 class="form-title">Contact me</h2>
+      <div class="contact-info">
+        <p>If you have an application you are interested in developing, a feature that you need build or a project that needs coding. I'd love to help with.</p>
+      </div>
+      
+      <form class="form-action" action="https://formspree.io/f/{form_id}" method="post">
```
Comment (mckent05) [inline index.html]: - Well done using the `post` method on the form, but you are required to sign up on the [formspree service](https://formspree.io/html) and integrate your form using the form_id

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @IvanLudick,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ 
- Your portfolio looks good 😄 😄 
- Your PR is professional 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
**OPTIONAL** - Please consider making your portfolio responsive for all mobile screens `<768px` this would ensure that your portfolio fits well and is responsive across all mobile screens. Please fix this 🙏 
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html (hunk 2 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @IvanLudick 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub-Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**